client/src/components/search/searchBar/search-bar.tsx

```diff
@@ -48,14 +48,14 @@ type searchBarPropType = {
   isSearchFocused?: boolean;
   t?: (label: string) => string;
```

Comment:
If you do end up making any of the above changes...

```suggestion
  t?: TFunction;
```